### ISSUE TITLE
Some details of the KiTS19 dataset.

### ISSUE THREAD

**Issue by byhwhite** [Jianf-Wang/GBDL#1]
Dear author:
Congratulations on your paper's accept! I have read your paper and do some experiments on KiTS19 dataset. Due to the unknown of the details of the dataset, the results of my experiments have no reference to compare. Hope you can help me:
1. If you use other released code for dataset's preprocess, could you tell me the code address?
2. If you use your own preprocess, I'd appreciate if you would release it. If there is some problems of releasing, could you just tell me the ROI shape of preprocess, and the shape of training batch?

**Comment by Jianf-Wang** [comment]
Hi, Thanks for your interest to this work. We did not do further pre-processing, just extracting '.png' images from the official files and using [-100, 250] HU range for them, and then, these images are directly used (these images are shared via Baidu disk). As for the ROI shape, we roughly cropped the 160 * 160 centre region, which is implemented in lines 440-444 in train.py, instead of a pre-processing step (this is a bit different from some previous works). This could be inaccurate, and a more precise ROI crop may further improve the performance.

**Comment by byhwhite** [comment]
I got it, thank you a lot! By the way, are the labeled set, unlabeled set and test set select in order? e.g. If label number is 16, take case 00000-00016 as labeled set, case 00017-00160 as unlabeled set, case 00161-00210 as test set.

**Comment by Jianf-Wang** [comment]
No, they are randomly set every time you run the training code. So, it is better to run experiments for several times to get a more objective result.

**Comment by byhwhite** [comment]
Got it! Thanks again!
